src/DynamoCoreWpf/Extensions/ViewExtensionDefinition.cs

```diff
@@ -14,5 +15,24 @@ internal class ViewExtensionDefinition
 
         [DataMember]
         public string TypeName { get; set; }
+
+        /// <summary>
+        /// Indicates path to xml file with extension definition.
+        /// I.e. *_ViewExtensionDefinition.xml
+        /// </summary>
+        public string ExtensionPath { get; set; }
+
+        /// <summary>
+        /// Gets path to extension assembly. 
+        /// This assembly should be in folder with the same name as assembly's name.
+        /// </summary>
+        public string AssemblyLocation
+        {
+            get
+            {
+                var path = Path.GetDirectoryName(ExtensionPath);
+                return Path.Combine(path, AssemblyName, AssemblyName + ".dll");                 
+            }
```

Comment:
Just for the record, the `AssemblyName` should be the absolute path to the assembly. Let's rename it to `AssemblyPath` for this and `ExtensionDefinition`.